packages/utilities/package.json

```diff
@@ -40,4 +42,4 @@
     "react": "^0.14 || ^15.0.1-0 || ^16.0.0-0",
```

Comment:
I've confirmed the following warning message when I build an application with react/react-dom 0.14.8 and prop-types 15.5.10.

>warning.js:45 Warning: Failed propType: Calling PropTypes validators directly is not supported by the `prop-types` package. Use `PropTypes.checkPropTypes()` to call them. Read more at http://fb.me/use-check-prop-types Check the render method of `AccountingView`.

React v0.14.9 works fine, without such a warning. Therefore I think we have to upgrade to 0.14.9.

Meanwhile, I have a concern too. React v0.14.9 does not appeared on its [CHANGELOG](https://github.com/facebook/react/blob/master/CHANGELOG.md). It seems a simple [mistake](https://twitter.com/dan_abramov/status/852193956322521089), but I suspect React v0.14.9 is not famous yet.

On the other hand, React v0.14.9 is a valid release version, we can install it by `^0.14.0`.

In conclusion, we should fix React version like below.

```
"peerDependencies": {
  "react": "^0.14.9 || ^15.0.1-0 || ^16.0.0-0",
  "react-dom": "^0.14.9 || ^15.0.1-0 || ^16.0.0-0"
}
```

How do you feel about it?